pep/app/pods/components/search/item/template.hbs

```diff
@@ -0,0 +1,45 @@
+<li class="mb-4 d-flex justify-content-between">
+    <div>
+        <a href="" {{on "click" (fn @openResultPreview @item)}}>
+            {{@item.documentRef}}
+        </a>
+        {{#if @item.kwicList}}
+            <ul class="font-size-secondary">
+                {{#each @item.kwicList as |searchHit|}}
+                    <li>{{html-safe searchHit}}</li>
+                {{/each}}
+            </ul>
+        {{/if}}
+    </div>
+    <div class="pl-2 d-flex justify-content-end align-items-start">
+        {{#if @item.relatedrx}}
+            <span class="mr-2">
+                <FaIcon @icon="books" class="p-0 "/>
+                <EmberTooltip
+                    @text={{t "search.item.relatedDocuments"}}
+                    @popperContainer="body"
+                    @side="left"
+                    @spacing={{5}}
+                    @effect="slide"
+                    @innerClass=""
+                />
+            </span>
+        {{/if}}
+        <Button
```

Comment:
these probably should use `Button:WithTooltip` (e.g. "Add to favorites" / "Remove from favorites" for accessibility)